packages/streams/README.md

```diff
@@ -0,0 +1,14 @@
+# @signalk/streams
+
+Used within the [Signal K](http://signalk.org) [Node Server](https://github.com/SignalK/signalk-server-node) to render the admin interface.
+
+This is its own package so when its installed the javascript asset files come compiled. It makes the build process from the git repo easier and more efficient.
```

Comment:
The code here is not involved in rendering the admin interface, but for streaming and converting input data in different formats.

The code is not compiled and is not more effective, but allows using the streams and their conversions independently outside the server.

Reply:
Hmm, was this supposed to go under the admin ui directory?

Reply:
The ./packages/streams packages was missing the README.md, so I've copied it from ./packages/server-admin-ui mostly for myself to remember how to do the npm stuff  to be able to develop it. Will replace the wording with your suggestions.